CMakeLists.txt

```diff
@@ -166,6 +166,39 @@ endif ()
 
 find_package(Git)
 
+if(GIT_FOUND)
```

Comment:
If git is not mandatory, maybe we should not include this check fail right away if git is not found?

Reply:
We can't make Git mandatory, it would prevent tar-ball builds. In fact, to a degree, our increased reliance on Git for builds caused this issue to begin with.

Reply:
The git information function behaves differently depending on whether Git (and the repository) is available or not. I updated the wording in the PR and the commit message to not call Git required.